packages/client/src/app/components/index.ts

```
@@ -29,6 +29,10 @@ import { QuestModal } from './modals/quests';
 import { RevealModal } from './modals/reveal';
 import { SettingsModal } from './modals/settings';
 import { TradingModal } from './modals/trading';
+import { AnimationStudio } from './modals/studio/AnimationStudio';
+// unused
+import { FundOperator } from './modals/FundOperator';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'FundOperator'. Consider to delete it or rename it.

'FundOperator' is defined here:


(lint/suspicious/noRedeclare)